### ISSUE TITLE
Tomster Compendium

### ISSUE THREAD

**Issue by ncwhitehead** [emberjs/website#1895]
I couldn't find an index of the various incarnations of Tomster. Having a page with all the versions of everyone's favorite hamster on it would be great, similar to the Octodex for Octocat. Please let me know if there is any interest in such a feature, and I would be happy to work on a pull request.

**Comment by wifelette** [comment]
Aloha @ncwhitehead! Hope you haven't lost enthusiasm for this... I got busy drowning in EmberConf :p EmberConf _has_ however resulted in 5-10 new Tomsters being designed as we speak, making this page even more useful. 

Assuming this is something we could use community help on, and even if not, best bet is a PR with a To Do list so we can knock off all the little tasks to get this done.

This is the last one we worked on in this section, and it went super well, with a ton of people jumping in to help: https://github.com/emberjs/website/pull/1904

Want to do the honors and we can start making more progress from there?
